### PR TITLE
Newbuttons

### PR DESCRIPTION
**Pull request by LAxelle**
Changes : 
- adding new buttons with corresponding functionality

### CODE REVIEW

File: src/controller/game_controller.py [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+    K_3,
+    K_4,
+    K_5,
+    K_6,
+    K_KP1,
+    K_KP2,
+    K_KP3,
+    K_KP4,
+    K_KP5,
+    K_KP6
```
Comment (Hboni) [inline src/controller/game_controller.py]: Add coma at the end of the line.

File: src/controller/game_controller.py [hunk 2]
```diff
@@ -18,7 +31,8 @@ def __init__(self, window):
 
         print("Enter in controller")
         self.window = window
-        self.humans_list = [Dealer()]
+        self.deck = Deck()
+        self.humans_list = [Player("Axelle",100)]
```
Comment (Hboni) [inline src/controller/game_controller.py]: Space after coma (PEP 8)

File: src/controller/game_controller.py [hunk 3]
```diff
@@ -65,7 +79,7 @@ def removePlayer(self, player_uuid) -> bool:
 
     def resetAllHumans(self):
         """Resetting all humans : no more player and new dealer"""
-        self.humans_list = [Dealer()]
+        self.humans_list = [Player("Axelle",100)]
```
Comment (Hboni) [inline src/controller/game_controller.py]: Space after coma (PEP 8).

File: src/controller/game_controller.py [hunk 4]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                        print(5)
+                        human.double(0)
+
+                    elif event.key == K_6 or event.key == K_KP6 :
+                        return False
 
+                elif event.type == pygame.MOUSEBUTTONDOWN:
+                    pos = pygame.mouse.get_pos()
+                    if self.view_game.quit_btn.isClicked(pos):
+                        return False
```
Comment (Hboni) [inline src/controller/game_controller.py]: No new line at the end (PEP 8)

**Review by Hboni — CHANGES_REQUESTED** [review]

File: src/controller/game_controller.py (hunk 1 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: Done

File: src/controller/game_controller.py (hunk 2 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: Done

File: src/controller/game_controller.py (hunk 3 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: Done

File: src/controller/game_controller.py (hunk 4 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: Done

**Review by Hboni — APPROVED** [review]

File: src/controller/game_controller.py [hunk 5]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                    if event.key == K_ESCAPE:
                         return False
-                    elif event.type == pygame.MOUSEBUTTONUP:
-                        pos = pygame.mouse.get_pos()
-                        if self.view_game.quit_btn.isClicked(pos):
-                            return False
-                    else:
-                        human.chooseAction()
+
+                    elif event.key == K_1 or event.key == K_KP1 :
```
Comment (deplanty) [inline src/controller/game_controller.py]: `event.key in [K_1, K_KP1]` -> Simpler and easier to add other conditions.

File: src/controller/game_controller.py [hunk 6]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                        print(1)
+                        for i in range(len(human.hands)):
+                            card = self.deck.getCard()
+                            human.addCard(card, i)
+
+                    elif event.key == K_2 or event.key == K_KP2 :
```
Comment (deplanty) [inline src/controller/game_controller.py]: Idem : use `in` instead of `or` (*)

File: src/controller/game_controller.py [hunk 7]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                        print(2)
+                        betAmount = 5
+                        handId = 0
+                        if len(human.hands) >= 2:
+                            print("You have " + len(human.hands) + "hands")
+                            handId = input("Hand number for bet : ")
+                        human.bet(int(betAmount), int(handId))
+
+                    elif event.key == K_3 or event.key == K_KP3 :
```
Comment (deplanty) [inline src/controller/game_controller.py]: Idem : *

File: src/controller/game_controller.py [hunk 8]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                        print(3)
                         state = False
-                        return
 
+                    elif event.key == K_4 or event.key == K_KP4 :
```
Comment (deplanty) [inline src/controller/game_controller.py]: Idem : *

File: src/controller/game_controller.py [hunk 9]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                        print(4)
+                        handId = 0
+                        if len(human.hands)>=2:
+                            print("You have " + len(human.hands) + "hands")
+                            handId = input("Hand number for bet : ")
+                        human.split(handId)
 
+                    elif event.key == K_5 or event.key == K_KP5 :
```
Comment (deplanty) [inline src/controller/game_controller.py]: Idem : *

File: src/controller/game_controller.py [hunk 10]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                        print(5)
+                        human.double(0)
+
+                    elif event.key == K_6 or event.key == K_KP6 :
```
Comment (deplanty) [inline src/controller/game_controller.py]: Idem : *

File: src/controller/game_controller.py [hunk 11]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+                    elif event.key == K_1 or event.key == K_KP1 :
+                        print(1)
+                        for i in range(len(human.hands)):
+                            card = self.deck.getCard()
+                            human.addCard(card, i)
+
+                    elif event.key == K_2 or event.key == K_KP2 :
+                        print(2)
+                        betAmount = 5
+                        handId = 0
```
Comment (deplanty) [inline src/controller/game_controller.py]: hand_id

File: src/controller/game_controller.py [hunk 12]
```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
...
+
+                    elif event.key == K_1 or event.key == K_KP1 :
+                        print(1)
+                        for i in range(len(human.hands)):
+                            card = self.deck.getCard()
+                            human.addCard(card, i)
+
+                    elif event.key == K_2 or event.key == K_KP2 :
+                        print(2)
+                        betAmount = 5
```
Comment (deplanty) [inline src/controller/game_controller.py]: bet_amount

File: src/views/view_game.py [hunk 13]
```diff
@@ -73,9 +73,55 @@ def init_game_btns(self):
             pos=(1100, 610),
             width=80,
             height=80,
-            text="Quit",
+            text="(6) Quit",
             background=(180, 180, 180),
         )
+
```
Comment (deplanty) [inline src/views/view_game.py]: Delete this empty line (too much blank lines)

**Review by deplanty — CHANGES_REQUESTED** [review]
PEP8 Nazi

File: src/controller/game_controller.py (hunk 5 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 10 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 9 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 8 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 7 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 6 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 11 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/controller/game_controller.py (hunk 12 above)
Comment (LAxelle) [reply src/controller/game_controller.py]: done

File: src/views/view_game.py (hunk 13 above)
Comment (LAxelle) [reply src/views/view_game.py]: done

**Review by deplanty — APPROVED** [review]

**Pull request merged**